tools/cassandra/README.md

```diff
@@ -6,19 +6,26 @@ How
 ---
 - Run make bins
 - You should see an executable `cadence-cassandra-tool`
+- Make sure you already created cadence and visibility keyspaces (see `keyspace.cql` in schema)
```

Comment:
This is a bit confusing. Aren't we creating this using the commands below? How will the user make sure they have already created the keyspaces.

Reply:
removed